### PR TITLE
Replace freenode IRC links with libera.chat equivalents

### PR DESCRIPTION
**Pull request by MoralCode**

### CODE REVIEW

File: projects/_posts/2020-03-01-teleirc.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Today, it is used by communities around the world.
 ## Resources and links
 
 * Community:
-    * [IRC channel](https://webchat.freenode.net/?channels=rit-lug-teleirc)
+    * [IRC channel](https://web.libera.chat/#rit-lug-teleirc)
```
Comment (Tjzabel) [inline projects/_posts/2020-03-01-teleirc.md:35]: So, TeleIRC is actually still on freenode for the time being. Everything looks great except this portion here.

**Review by Tjzabel — CHANGES_REQUESTED** [review]
Looks great besides the TeleIRC portion. The project is still on freenode.

File: projects/_posts/2020-03-01-teleirc.md (hunk 1 above)
Comment (MoralCode) [reply projects/_posts/2020-03-01-teleirc.md:35]: oh ok. ill revert this section

**Comment by Tjzabel** [comment]
@MoralCode Thanks! Do you mind fixing the CI as well? There's a simple 404'd link to remove in a post. (Can do that in another PR if you really want to for more hacktoberfest contributions).

**Comment by MoralCode** [comment]
I cant seem to find an alternative, non-404 version of that link on archive.org or unicefs blog archives. Maybe @jwflory knows where the current canonical version of this page is?

**Comment by Tjzabel** [comment]
Personally, I was thinking you could just keep the text but remove the dead
link from it.

**Comment by MoralCode** [comment]
good point, i commented out the link part to preserve the fact that there used to be a link there. It would still be ideal to fix the link longer term though.

**Review by Tjzabel — APPROVED** [review]
LGTM

**Comment by MoralCode** [comment]
Site finally got redeployed with these changes. I also realized I missed some spots and so i fixed those too (directly on main :smiling_imp: )

**Comment by MoralCode** [comment]
also several other links were 404ing during the build but seemed fine in my browser (i think twitter is mow blocking automated traffic more aggressively). made it so that CI jobs dont fail because of broken links because we cant control the uptime of every site we link to

**Pull request merged**